--- a/SamTFE/Sources/Engine/Graphics/Gfx_Vulkan.cpp	2024-02-14 23:09:04.922212716 +0300
+++ b/SamTFE/Sources/Engine/Graphics/Gfx_Vulkan.cpp	2024-02-14 23:09:16.938635079 +0300
@@ -169,7 +169,7 @@
   appInfo.applicationVersion = VK_MAKE_VERSION(1, 10, 6);
   appInfo.pEngineName = "Serious Engine 1";
   appInfo.engineVersion = VK_MAKE_VERSION(1, 10, 6);
-  appInfo.apiVersion = VK_API_VERSION_1_2;
+  appInfo.apiVersion = VK_API_VERSION_1_0;
 
 
   VkInstanceCreateInfo instanceInfo = {};
--- a/SamTSE/Sources/Engine/Graphics/Gfx_Vulkan.cpp	2024-02-14 23:09:04.922212716 +0300
+++ b/SamTSE/Sources/Engine/Graphics/Gfx_Vulkan.cpp	2024-02-14 23:09:16.938635079 +0300
@@ -169,7 +169,7 @@
   appInfo.applicationVersion = VK_MAKE_VERSION(1, 10, 6);
   appInfo.pEngineName = "Serious Engine 1";
   appInfo.engineVersion = VK_MAKE_VERSION(1, 10, 6);
-  appInfo.apiVersion = VK_API_VERSION_1_2;
+  appInfo.apiVersion = VK_API_VERSION_1_0;
 
 
   VkInstanceCreateInfo instanceInfo = {};
